src/bun.js/api/ffi.zig

```diff
@@ -2064,7 +2080,7 @@ pub const FFI = struct {
         function = 17,
         napi_env = 18,
         napi_value = 19,
-
+        buffer = 20,
         pub const max = @intFromEnum(ABIType.napi_value);
```

Comment:
```suggestion
        pub const max = max_abi_type: {
            var current_max = std.math.minInt(i32);
            for (@typeInfo(ABIType).Enum.fields) |f| {
                current_max = @max(current_max, f.value);
            }
            break :max_abi_type current_max;
        };
```

or, could we `@cImport` these?